test/typing/pass/math_ops.py

```diff
@@ -0,0 +1,327 @@
+# flake8: noqa
+import torch
+import math
+
+a = torch.randn(4)
+b = torch.randn(4)
+t = torch.tensor([-1, -2, 3], dtype=torch.int8)
+
+# abs/absolute
+torch.abs(torch.Tensor([-1, -2, 3]))
+torch.absolute(torch.Tensor([-1, -2, 3]))
```

Comment:
Please don't use `torch.Tensor`, it's the deprecated constructor and we just got rid of it everywhere. use `torch.tensor` instead